charts/artifact-registry/chart/templates/deployment-image-registry.yaml

```
@@ -0,0 +1,61 @@
+{{with .Values.imageRegistry}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: image-registry
+  name: image-registry
+spec:
+  selector:
+    matchLabels:
+      app: image-registry
+  replicas: {{.replicaCount}}
+  template:
+    metadata:
+      labels:
+        app: image-registry
+    spec:
+      {{- with .affinity}}
+      affinity:
+      {{toYaml . | indent 8}}
+      {{- end }}
+      containers:
+      - image: "{{ .image.repository }}:{{ .image.tag }}"
+        imagePullPolicy: {{.image.pullPolicy}}
+        name: image-registry
+        resources:
+        {{- toYaml .resources | nindent 12}}
+        readinessProbe:
+          exec:
+            command:
+            - nc
+            - -z
+            - -v
+            - "localhost"
+            - "{{.service.port}}"
```

Can we just use a normal HTTP probe here? Ex:
```suggestion
          httpGet:
            port: {{.service.port}}
```